modules/virt-jumbo-frames-vm-pod-nw.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc              
+
+:_content-type: CONCEPT                                            
+[id="virt-jumbo-frames-vm-pod-nw_{context}"]                                   
+= About jumbo frames support          
+
+When using the OVN-Kubernetes CNI plug-in, you can send unfragmented jumbo frame packets between two virtual machines (VMs) that are connected on the default pod network. Jumbo frames have a maximum transmission unit (MTU) value greater than 1500 bytes.
+
+The VM automatically gets the MTU value of the cluster network, set by the cluster administrator, in one of the following ways:
+
+* `libvirt`: If the guest OS has the latest version of the VirtIO driver that can interpret incoming data via a PCI config register in the emulated device. 
```

Comment:
Should PCI be spelled out before it's used as an acronym?

Reply:
I don't think it is necessary here but will spell it out for consistency with other acronyms that are used in the module